### DISCUSSION TITLE
open channel from lnd: unexpected channel type

### DISCUSSION THREAD

**Discussion by C-Otto** [ACINQ/eclair#2034] [Q&A] [answered]
Hi, I run lnd 0.14 (master) and have troubles opening a channel to ACINQ (despite already having two open channels). lnd complains about "unexpected channel type" (https://github.com/lightningnetwork/lnd/blob/7f656a25500d246405c3fd51ea741cac1874722e/funding/manager.go#L1589). Any idea what might be the cause of this? My invocation attempt:

```
lncli openchannel --node_key 03864ef025fde8fb587d989186ce6a4a186895ee44a926bfc370e2c366597a3f8f --local_amt 49000000 --sat_per_vbyte 1 --min_confs 0 
[lncli] rpc error: code = Unknown desc = received unexpected channel type
```

**Comment by t-bast** [comment, marked as answer]
This is probably because lnd is trying to open an `anchor_outputs_zero_fee_htlc_tx` channel, whereas our node did not activate this feature.
So our node replies with the `channel_type` it would accept, but lnd seems to truncate our error message.
If you look at your logs, you should see what `channel_type` we expect.
Then hopefully the lnd cli lets you specify the `channel_type` you want to use in `openchannel`.

In all cases, this is an issue that should be opened on the lnd repository IMHO.

**Reply by C-Otto** [reply]
Fixed in lnd 0.14.1

**Comment by C-Otto** [comment]
Yeah, it's an lnd issue. Seems to be related to this: https://github.com/lightningnetwork/lnd/issues/5890
